### ISSUE TITLE
An error occurred: 'MyPlexAccount' object has no attribute 'watchlist'

### ISSUE THREAD

**Issue by profucius** [wajeht/plex-watchlist-cleaner#1]
Hi there. I found your repo when searching around for a way to batch-remove all items from my Plex Watchlist. It seems to me that your python script can accomplish this, but if I've misunderstood please let me know.

I followed the instructions in your readme and ran the script. It gives me an error (title). I'm not sure how to proceed from here. I can provide any debug details if you want them. If there is a way to fix this I'd be very happy. Thank you.

**Comment by wajeht** [comment]
I just tested it and still works. What exactly are you having problems with?

<img width="1022" alt="image" src="https://github.com/wajeht/plex-watchlist-cleaner/assets/58354193/b575ed62-90f5-46ac-872a-a81e28057459">

**Comment by profucius** [comment]
Interesting. Could it be related to Python version? I'm using Python 3.11.0, on Windows 11. I downloaded the repo as a zip, extracted and opened a cmd inside the folder. Python is in PATH.

![image](https://github.com/wajeht/plex-watchlist-cleaner/assets/18254607/ba25790f-9127-44e7-af8f-f693c9d0d1d5)

Run the command `python index.py` and it works for about 1 second, then outputs:

![image](https://github.com/wajeht/plex-watchlist-cleaner/assets/18254607/9213763f-7560-47c9-9265-16e364548f72)

**Comment by profucius** [comment]
Ok I just re-read your Readme instructions, I think I skipped "activating" the virtual env. I started all over, fresh repo download, recreated the .env file with my credentials. Then ran `python -m venv env` with no cmd result. Then ran `env\Scripts\activate` with no cmd result. Then ran `python index.py` and I got a new error:

![image](https://github.com/wajeht/plex-watchlist-cleaner/assets/18254607/ed211d9e-cfb1-414b-878f-af7fb6b6da8e)

**Comment by wajeht** [comment]
Alright my friend. try on this GUI I just built. 

`https://plex-watchlist-cleaner.jaw.dev/`

code is also open source.

`https://github.com/wajeht/plex-watchlist-cleaner-ui`

<img width="1230" alt="image" src="https://github.com/wajeht/plex-watchlist-cleaner/assets/58354193/1e59f95e-09b4-446a-bc9f-48ee63913e4d">

it will give you all the deleted items in watch list

**Comment by profucius** [comment]
This looks useful for people planning to achieve the solution manually via UI. However, I was planning on using a command-line approach to this solution, since I am integrating it into a larger automation I am writing. 

My goal is to clear my Plex watchlist on a nightly basis, because I have my Plex watchlist auto sync to my Trakt watchlist using a different script, and so on a daily basis (via cmd) I will have that sync, and then I planned to use your script to clear my Plex watchlist automatically after.

Question about the original script: Can I modify it to not require a virtual env, so that it will use my main site-packages? My thought is that the script will work for me if I can either 1) Get the virtual env created properly, or 2) Drop the virtual env and use my main one.

**Comment by wajeht** [comment]
use this. 

```bash
$ python index.py username password
```

```python
import sys
import json
from plexapi.myplex import MyPlexAccount

def main():
    if len(sys.argv) < 3:
        print("Missing username or password")
        return

    username = sys.argv[1]
    password = sys.argv[2]

    try:
        account = MyPlexAccount(username, password)
        watchlist = account.watchlist()

        removed_items = []

        for item in watchlist:
            removed_items.append(item.title)
            account.removeFromWatchlist(item)

        print(removed_items)

    except Exception as e:
        error_message = str(e)
        print(error_message)

if __name__ == '__main__':
    main()
```

**Comment by profucius** [comment]
I appreciate you trying to help me out here. For some reason it keeps failing on me. 🤦‍♂️ 

I edited the index.py file with your script above, then ran the command with my user:pass in the command as instructed. I am still getting no response.

I even tried all of this on a totally different machine with a completely clean install of Python 3.12 and installed requirements to it. This is what I get:

![image](https://github.com/wajeht/plex-watchlist-cleaner/assets/18254607/e74928a4-3531-411b-a71b-27ba03dd6cae)

It's the same response I get when I used the original venv method, the script returns nothing.

Is there a log output anywhere that I can see how/why this fails? I looked in a few folders but I didn't find a log, but perhaps Python itself outputs a log? Or maybe a verbose arg would do so? Sorry for the trouble, at this point I just cant tell if its me or python.

**Comment by wajeht** [comment]
if you read the code, it will append all the removed items into a list and print them out at the end of the task. 

if no watchlist was removed, the list will not contain anything.

```python
[]
```

if anything was removed, it will look like this.

```python
["Spider-Man", "Barbie"]
```

**Comment by profucius** [comment]
I just tested again and it worked! It occurred to me that I was juggling too many things at once while diagnosing the original issue, and I had manually emptied the watchlist to start over, but didn't add anything back to the watchlist again. I thought the output would say it found nothing if it hadn't, so I was assuming the script wasn't working. 

My mistake, and apologies for the mixup. Thanks again for helping me iron things out, this script will be a big help to me.
